src/main/java/org/jabref/gui/WelcomeTab.java

```diff
@@ -0,0 +1,274 @@
+        openLibraryLink.setOnAction(e -> new OpenDatabaseAction(tabContainer, preferences, aiService, dialogService,
+                stateManager, fileUpdateMonitor, entryTypesManager, undoManager, clipBoardManager,
+                taskExecutor).execute());
+
+        return createVBoxContainer(startLabel, newLibraryLink, openLibraryLink);
+    }
+
+    private VBox createWelcomeRecentBox() {
+        Label recentLabel = new Label(Localization.lang("Recent"));
+        recentLabel.getStyleClass().add("welcome-header-label");
+
+        recentLibrariesBox.setAlignment(Pos.TOP_LEFT);
+        updateWelcomeRecentLibraries();
+
+        fileHistoryMenu.getItems().addListener((ListChangeListener<MenuItem>) change -> updateWelcomeRecentLibraries());
+
+        return createVBoxContainer(recentLabel, recentLibrariesBox);
+    }
+
+    private void updateWelcomeRecentLibraries() {
+        if (fileHistoryMenu.getItems().isEmpty()) {
+            displayNoRecentLibrariesMessage();
+            return;
+        }
+
+        recentLibrariesBox.getChildren().clear();
+        fileHistoryMenu.disableProperty().unbind();
+        fileHistoryMenu.setDisable(false);
+
+        for (MenuItem item : fileHistoryMenu.getItems()) {
+            Hyperlink recentLibraryLink = new Hyperlink(item.getText());
+            recentLibraryLink.getStyleClass().add("welcome-hyperlink");
+            recentLibraryLink.setOnAction(item.getOnAction());
+            recentLibrariesBox.getChildren().add(recentLibraryLink);
+        }
+    }
+
+    private void displayNoRecentLibrariesMessage() {
+        recentLibrariesBox.getChildren().clear();
+        Label noRecentLibrariesLabel = new Label(Localization.lang("No recent libraries"));
+        noRecentLibrariesLabel.getStyleClass().add("welcome-no-recent-label");
+        recentLibrariesBox.getChildren().add(noRecentLibrariesLabel);
+
+        fileHistoryMenu.disableProperty().unbind();
+        fileHistoryMenu.setDisable(true);
+    }
+
+    private VBox createVBoxContainer(Node... nodes) {
+        VBox box = new VBox(5);
+        box.setAlignment(Pos.TOP_LEFT);
+        box.getChildren().addAll(nodes);
+        return box;
+    }
+
+    private VBox createFooter() {
+        Label communityLabel = createFooterLabel(Localization.lang("Community"));
+
+        HBox iconLinksContainer = createIconLinksContainer();
+        HBox textLinksContainer = createTextLinksContainer();
+        HBox versionContainer = createVersionContainer();
+
+        VBox footerBox = new VBox(10);
+        footerBox.setAlignment(Pos.CENTER);
+        footerBox.getChildren().addAll(communityLabel, iconLinksContainer, textLinksContainer, versionContainer);
+        footerBox.setPadding(new Insets(10, 0, 10, 0));
+        footerBox.getStyleClass().add("welcome-footer-container");
+
+        return footerBox;
+    }
+
+    private Label createFooterLabel(String text) {
+        Label label = new Label(Localization.lang(text));
```

Comment:
```suggestion
        Label label = new Label(text);
```

Localiziation was done at the caller